iina/fr.lproj/PrefSubViewController.strings

```diff
@@ -0,0 +1,123 @@
+
+/* Class = "NSMenuItem"; title = "Subtitles containing media filename"; ObjectID = "1vo-so-iaM"; */
+"1vo-so-iaM.title" = "Sous-Titres contenant titre de fichier média";
+
+/* Class = "NSMenuItem"; title = "Shooter.cn"; ObjectID = "5Bo-HI-z0D"; */
+"5Bo-HI-z0D.title" = "Shooter.cn";
+
+/* Class = "NSBox"; title = "Border"; ObjectID = "5CZ-8l-KYJ"; */
+"5CZ-8l-KYJ.title" = "Bordure";
+
+/* Class = "NSTextFieldCell"; title = " X:"; ObjectID = "7NC-Ih-gMB"; */
+"7NC-Ih-gMB.title" = " X:";
+
+/* Class = "NSTextFieldCell"; title = "sans-serif"; ObjectID = "Bjp-HJ-i7V"; */
+"Bjp-HJ-i7V.title" = "sans-serif";
+
+/* Class = "NSTextFieldCell"; title = "Prefered language:"; ObjectID = "Chj-YP-eFV"; */
+"Chj-YP-eFV.title" = "Langue Préferrée:";
+
+/* Class = "NSMenuItem"; title = "Center"; ObjectID = "HCH-sL-fT2"; */
+"HCH-sL-fT2.title" = "Centre";
+
+/* Class = "NSMenuItem"; title = "Left"; ObjectID = "Hpt-KA-McW"; */
+"Hpt-KA-McW.title" = "Gauche";
+
+/* Class = "NSTextFieldCell"; title = " X:"; ObjectID = "Jvu-MV-7vD"; */
+"Jvu-MV-7vD.title" = " X:";
+
+/* Class = "NSTextFieldCell"; title = "Download subtitles from:"; ObjectID = "MiO-eP-c1f"; */
+"MiO-eP-c1f.title" = "Télécharger les Sous-Titres de:";
+
+/* Class = "NSTextFieldCell"; title = "Text subtitles:"; ObjectID = "Mte-Lb-do4"; */
+"Mte-Lb-do4.title" = "Sous-Titres textes:";
+
+/* Class = "NSMenuItem"; title = "Bottom"; ObjectID = "ND2-m7-ZET"; */
+"ND2-m7-ZET.title" = "Bas";
+
+/* Class = "NSTextFieldCell"; title = "Size:"; ObjectID = "OL9-2M-Zb5"; */
+"OL9-2M-Zb5.title" = "Grandeur:";
+
+/* Class = "NSTextFieldCell"; title = "Other:"; ObjectID = "PZ2-pZ-hHa"; */
+"PZ2-pZ-hHa.title" = "Autre:";
+
+/* Class = "NSMenuItem"; title = "All subtitles in the same folder"; ObjectID = "Qcl-0x-3hh"; */
+"Qcl-0x-3hh.title" = "Tous les Sous-Titres dans le même dossier";
+
+/* Class = "NSTextFieldCell"; title = "Font:"; ObjectID = "RiL-Rv-eke"; */
+"RiL-Rv-eke.title" = "Police:";
+
+/* Class = "NSTextFieldCell"; title = "Background:"; ObjectID = "SLQ-wt-k6Y"; */
+"SLQ-wt-k6Y.title" = "Fond d'écran:";
```

Comment:
"Fond d'écran" is a wallpaper; what we're looking for here is "arrière-plan" I think :)